fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java

```diff
@@ -1015,10 +1016,10 @@ public void closeWriter(TBrokerFD fd) throws UserException {
         FSDataOutputStream fsDataOutputStream = ioStreamManager.getFsDataOutputStream(fd);
         synchronized (fsDataOutputStream) {
             try {
-                fsDataOutputStream.flush();
+                fsDataOutputStream.hsync();
```

Comment:
Why change to hsync?

Reply:
To make sure the hdfs file is sync to the disk.
Especially when the disk is almost full, the hsync can detect and throw exception.

Reply:
https://github.com/StarRocks/starrocks/pull/10713 @xiaoyong-z This pull request already add the log

Reply:
this pr adds log in FE, 
https://github.com/StarRocks/starrocks/pull/10713 adds log in broker